service/src/main/resources/db/changelog.xml

```diff
@@ -1,3 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+  <property name="uuid_type" value="uuid" dbms="postgresql"/>
```

Comment:
How did you know what files here to modify? (again, no actions necessary, just tryign to understand the backend a bit more)

Reply:
We're using [liquibase](https://liquibase.org/) to manage changes to our postgres database. To make a new change, I created a "changeset" file specifying the operations I wanted to do. Liquibase will apply all of the changesets in `changelog.xml` and create a db table to track what was already run.